lib/galaxy/tool_util/deps/mulled/util.py

```diff
@@ -79,14 +82,32 @@ def mulled_tags_for(namespace, image, tag_prefix=None):
 
 def split_tag(tag):
     """Split mulled image name into conda version and conda build."""
```

Comment:
This is also not the image name but the image tag. Can you please change this older message.

Reply:
Done, does it look ok now @bgruening ?

Reply:
Yes :) Thanks!